gui/admin-gui/src/main/java/com/evolveum/midpoint/web/page/admin/cases/PageCaseWorkItems.java

```diff
@@ -147,22 +151,54 @@ private ObjectQuery createQuery() throws SchemaException, ObjectNotFoundExceptio
         } else {
             // not authorized to see all => sees only allocated to him (not quite what is expected, but sufficient for the time being)
             query = QueryUtils.filterForAssignees(q, SecurityUtils.getPrincipalUser(),
-                    OtherPrivilegesLimitationType.F_APPROVAL_WORK_ITEMS)
-                    .and().item(CaseWorkItemType.F_CLOSE_TIMESTAMP).isNull().build();
+                    OtherPrivilegesLimitationType.F_APPROVAL_WORK_ITEMS).build();
```

Comment:
`OtherPrivilegesLimitationType.F_CASE_MANAGEMENT_WORK_ITEMS)` should be here instead.